addons/fleet/models/fleet_vehicle.py

```diff
@@ -89,7 +89,11 @@ def _compute_manager_id(self):
     net_car_value = fields.Float(string="Purchase Value", help="Purchase value of the vehicle")
     residual_value = fields.Float()
     plan_to_change_car = fields.Boolean(related='driver_id.plan_to_change_car', store=True, readonly=False)
+    plan_to_change_bike = fields.Boolean(related='driver_id.plan_to_change_bike', store=True, readonly=False)
```

Comment:
This field should be defined on a migration script (Initialize the colum + set the value).